cibuildwheel/linux.py

```diff
@@ -117,10 +117,20 @@ def check_all_python_exist(
     *, platform_configs: Iterable[PythonConfiguration], container: OCIContainer
 ) -> None:
     exist = True
+    has_manylinux_interpreters = True
     messages = []
+
+    try:
+        # use capture_output to keep quiet
+        container.call(["manylinux-interpreters", "--help"], capture_output=True)
+    except subprocess.CalledProcessError:
+        has_manylinux_interpreters = False
```

Comment:
```suggestion
    with contextlib.suppress(subprocess.CalledProcessError):
        # use capture_output to keep quiet
        container.call(["manylinux-interpreters", "--help"], capture_output=True)
        has_manylinux_interpreters = True
```

(also add contextlib import above)